app/bundles/EmailBundle/EventListener/BroadcastSubscriber.php

```diff
@@ -34,9 +34,9 @@ public function onBroadcast(ChannelBroadcastEvent $event): void
         // Get list of published broadcasts or broadcast if there is only a single ID
         $emails = $this->model->getRepository()->getPublishedBroadcasts($event->getId());
 
-        while (false !== ($email = $emails->next())) {
-            $emailEntity                                            = $email[0];
-            [$sentCount, $failedCount, $failedRecipientsByList]     = $this->model->sendEmailToLists(
+        foreach ($emails as $email) {
```

Comment:
Isn't this loading all the emails to memory at once? From the previous implementation is is obvious (hopefully) that it keeps just 1 email in the memory.

Reply:
It's a replacement for `\Doctrine\ORM\AbstractQuery::iterate`, which was used before.